### PR TITLE
vimPlugins.telescope-fzy-native-nvim: init at 2021-01-02

### PR DESCRIPTION
**Pull request by jlesquembre**
###### Motivation for this change
Add https://github.com/nvim-telescope/telescope-fzy-native.nvim

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108264` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>spacevim</li>
<li>vimPlugins.telescope-fzy-native-nvim</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108264` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>spacevim</li>
<li>vimPlugins.telescope-fzy-native-nvim</li>
</ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by jlesquembre** [comment]
@SuperSandro2000 thanks for reviewing. Is it ready to be merged as is it?

File: pkgs/misc/vim-plugins/overrides.nix [hunk 1]
```diff
@@ -661,6 +661,32 @@ self: super: {
...
+          stdenv.mkDerivation {
+            name = "fzy-lua-native";
+            src = "${old.src}/${fzy-lua-native-path}";
+            # remove pre-compiled binaries
+            preBuild = "rm -rf static/*";
+            installPhase =
+              ''
+                install -Dm 444 -t $out/static static/*
+                install -Dm 444 -t $out/lua lua/*
+              '';
```
Comment (SuperSandro2000) [inline pkgs/misc/vim-plugins/overrides.nix:679]:
```suggestion
            installPhase = ''
              install -Dm 444 -t $out/static static/*
              install -Dm 444 -t $out/lua lua/*
            '';
```

**Review by SuperSandro2000 — APPROVED** [review]
please squash the commits so that we do not forget that.

**Comment by jlesquembre** [comment]
@SuperSandro2000 done

**Comment by jlesquembre** [comment]
Fixed the merge conflict
@SuperSandro2000 Since you already approved the changes, can we get it merged?

**Pull request merged**